manim/mobject/geometry.py

```diff
@@ -711,7 +711,20 @@ def set_opacity(self, opacity, family=True):
 
 
 class DashedLine(Line):
-    """A dashed Line.
+    """A dashed :class:`Line`.
+
+    Parameters
+    ----------
+    args : Any
+        Arguments to be passed to :class:`Line`
+    dash_length : :class:`float`, optional
+        The length of each individual dash of the line.
+    dash_spacing : Optional[:class:`float`]
+        No purpose.
```

Comment:
Actually, it is used in the `calculate_positive_space_ratio` method -- but that method actually isn't used anywhere.

I wonder whether it makes sense to keep the method around.

Reply:
Although it technically appears in `calculate_positive_space_raito`, I don't think that function can ever be called since `dash_spacing` defaults to None and isn't modified anywhere. It just errors everytime.

Reply:
It should be deprecated, or fixed, but either way it should be done in a separate PR.